src/tour.ts

```diff
@@ -81,11 +80,60 @@ export interface TourOptions {
   useModalOverlay?: boolean;
 }
 
+const SHEPHERD_DEFAULT_API = 'https://shepherdpro.com';
+const SHEPHERD_USER_ID = 'shepherdPro:userId';
+
+export class ShepherdPro extends Evented {
+  public activeTour: Tour | null | undefined;
+  public apiKey: string | undefined;
+  public apiPath: string | undefined;
+  public dataRequester: DataRequest | undefined;
+
+  init(apiKey: string, apiPath?: string) {
```

Comment:
```suggestion
  init(apiKey?: string, apiPath?: string) {
```

I think if we are expecting people to sometimes pass undefined and throwing an error if so, we should allow undefined with `?`